pkg/shell/indexes.js

```diff
@@ -583,6 +584,41 @@ function MachinesIndex(index_options, machines, loader, mdialogs) {
         update_active_machine(machine ? machine.address : null);
     }
 
+    function update_docs(machine, state, compiled) {
+        const item = compiled.items[state.component];
+        const docs_items = document.getElementById("navbar-docs-items");
+        docs_items.innerHTML = "";
+
+        function create_item(name, url) {
+            const el_li = document.createElement("li");
+            const el_a = document.createElement("a");
+            const el_icon = document.createElement("i");
+            el_icon.className = "fa fa-external-link fa-xs";
+            el_a.setAttribute("translate", "yes");
+            el_a.setAttribute("href", url);
+            el_a.setAttribute("target", "blank");
+            el_a.setAttribute("rel", "noopener noreferrer");
+
+            el_a.appendChild(document.createTextNode(name));
+            el_a.appendChild(el_icon);
+
+            el_li.appendChild(el_a);
+            docs_items.appendChild(el_li);
+        }
+
+        const os_release = JSON.parse(window.localStorage['os-release'] || "{}");
+        let os_name = os_release.NAME;
+        if (os_name === "Red Hat Enterprise Linux")
+            os_name = "RHEL";
```

Comment:
I'm not sure if that is legit -- in the official docs it's always spelled out.

Reply:
Right, it is better to spell it out, so I removed this part.